### DISCUSSION TITLE
How to change the ensure_installed list?

### DISCUSSION THREAD

**Discussion by PCBismarck** [LazyVim/LazyVim#2584] [Q&A] [answered]
I copy my lazyvim config file from another  computer, and I dont want to install some of the lsp installed on previous computer, so i want to how to set the ensure_installed list manually

now each time when i edit my file, it always try to install lot of lsp, and will result to this:
![failed](https://github.com/LazyVim/LazyVim/assets/93439675/a09e28b4-9b5d-48b7-b042-419b1b4ac76a)

this is my ensure_installed list in setting, i dont know how to change it
![mason-setting](https://github.com/LazyVim/LazyVim/assets/93439675/e7487e63-1b49-4f6e-96da-552e2235bcbf)

and, these are my tries to config it, but it dont work
![mytry](https://github.com/LazyVim/LazyVim/assets/93439675/e9844ad1-e92c-4bd7-b624-80ad967d8a45)

**Comment by 0x2E** [comment, marked as answer]
This should overwrite entire `ensure_installed`:

```lua
return {
	"williamboman/mason.nvim",
	opts = function(_, opts)
		opts.ensure_installed = {
			"clangd",
		}
	end,
}

```

**Reply by PCBismarck** [reply]
thanks, it works for me!
